src/app/lib/config/services/portuguese.js

```diff
@@ -34,6 +34,7 @@ const service = {
     manifestPath: '/manifest.json',
     swPath: '/sw.js',
     frontPageTitle: 'Notícias, vídeos, análise e contexto em português',
```

Comment:
Can you please also add `passportHomes: ['BBCScotland'],` to the `src/app/lib/config/services/scotland.js` page? It'll enable the Scotland articles to render on the Test environment instead of 404'ing `https://www.test.bbc.co.uk/scotland/articles/czwj5l0n210o` 

Ideally I'd rather the in the config be `passportHomeOverride` to be clear what it should be matching.

Reply:
How do I test this locally? I get an article at `http://localhost.bbc.com:7080/scotland/articles/czwj5l0n210o` even without adding the override.